### DISCUSSION TITLE
`dotnet watch run` reloads on cshtml / liquid file changes

### DISCUSSION THREAD

**Discussion by jptissot** [OrchardCMS/OrchardCore#7535] [Q&A]
I have not been able to figure this one out. I would like the project to recompile only when modifying source files but the watcher recompiles anytime I modify the look and feel of a view. 
I tried removing the files by using the example [here](https://docs.microsoft.com/en-us/aspnet/core/tutorials/dotnet-watch?view=aspnetcore-3.1#customize-files-list-to-watch) but it does not work. Maybe something to do with how we embed files ? Do you have any ideas @jtkech

**Comment by jtkech** [comment]
Try to reference directly from your app the razor runtime compilation package (i forgot the exact name), normally we don't need it as we reference it elsewhere, but maybe some targets files that will just set the right options

**Comment by netwavebe** [comment]
I think this is a fairly recent change. I used to change views and hit F5 in the browser to get an update without compilation. A week ago I noticed it's now compiling.
